Simultaneous control of Feedback 360 examples update

dmfgd1 opened the pull request:
- Reworked the Read.ino & Rotate.ino samples for the 2.0.0 update.
   - Rotate.ino now utilizes coherent target switching.
   - Could be refined through the introduction of a target data type, consisting of a target ID and an associated rotation. 
- All examples were tested on the Arduino Nano IoT 33
    - The code has been updated with the interrupt check to reflect this. Could be separated into another pull request if desired.
    
Please let me know if there are any questions. Thank you so much for this fantastic library!

HyodaKazuaki reviewed (approved):
LGTM!

Thank you for testing example sketch and identifying the issues.
Would you mind updating the 'Arduino Nano' column to "supported" in [SUPPORTED.md](https://github.com/HyodaKazuaki/Parallax-FeedBack-360-Servo-Control-Library-4-Arduino/blob/master/SUPPORTED.md)?

dmfgd1 commented:
Thank you for reviewing my pull request @HyodaKazuaki !

### SUPPORT.md
- I have updated the [SUPPORT.md ](https://github.com/HyodaKazuaki/Parallax-FeedBack-360-Servo-Control-Library-4-Arduino/blob/master/SUPPORTED.md) per your request. 
- To clarify, I used an Arduino Nano IoT 33 to test my code, which has some substantial differences to the traditional Arduino Nano (e.g. different processor, varying interrupt pins, etc.). As such, I have added a column specifically for the Nano IoT 33. 

### Further changes
- Since opening this pull request, I have begun making further contributions to the repository, including:
    - Implementing varying forms of velocity control to the servo motors, enabling them to perform simultaneously although with varying speed.
    - Adding helper functions to simplify commonly implemented logic, particularly for multi-motor use cases.
    - Expanding inline documentation to clarify existing and soon-to-be implemented functionality. Potential frustrations have been addressed by warning developers of possibly undesired behaviors, alongside providing recommended implementations and examples.
    
Would you like me to include the new changes in the current pull request, or open a new one?
Let me know if you have any questions or suggestions!
